app/views/candidate_mailer/_new_flow_chase_reference.text.erb

```diff
@@ -11,4 +11,4 @@ You can sign into your account to:
 
 [Sign into your account](<%= candidate_magic_link(@application_form.candidate) %>).
 
-<%= @application_form.application_choices.pending_conditions.first.provider.name %> must check your references before they can confirm your place on the course. Contact them if you need help getting references or choosing who to ask.
\ No newline at end of file
+<%= @application_form.application_choices.pending_conditions.first&.provider&.name %> must check your references before they can confirm your place on the course. Contact them if you need help getting references or choosing who to ask.
```

Comment:
Is this just a short term fix for the tests? Longer term it’d be good to make sure the provider name never returns nil somehow, as this’ll end up quite an odd sentence...

Reply:
yes, a short term fix for the tests. resolving this will be part of the post-references cleanup operation.

Reply:
👍